### PR TITLE
agnhost: merge registry.k8s.io/stress:v1 (github.com/vishh/stress)

### PR DESCRIPTION
**Pull request by AkihiroSuda**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

Merge [vishh/stress@eab4e3384bcad9899b8b801b4a1917a758e97d96](https://github.com/vishh/stress/tree/eab4e3384bcad9899b8b801b4a1917a758e97d96) into agnhost.

Old usage: `stress -mem-alloc-size 12Mi -mem-alloc-sleep 10s -mem-total 4Gi`
New usage: `agnhost stress --mem-alloc-size 12Mi --mem-alloc-sleep 10s --mem-total 4Gi`

Will replace `registry.k8s.io/stress:v1` used here:
https://github.com/kubernetes/kubernetes/blob/244fbf94fd736e94071a77a8b7c91d81163249d4/test/e2e_node/eviction_test.go#L1080-L1086

#### Which issue(s) this PR fixes:

This is a part of the steps to migrate from legacy Schema 1 images
- #123146

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by AkihiroSuda** [comment]
cc @vishh @dims @BenTheElder

File: third_party/forked/vishhstress/vishhstress.go
Comment (dims) [inline third_party/forked/vishhstress/vishhstress.go:1]: let's please name this `third_party/forked/vishhstress/stress.go` and drop the extra `vishh` in the file name

Comment (AkihiroSuda) [reply third_party/forked/vishhstress/vishhstress.go:1]: done

**Comment by dims** [comment]
@AkihiroSuda do you want to update test/images/agnhost/VERSION (for a new image to be built) in this PR itself?

**Comment by AkihiroSuda** [comment]
> @AkihiroSuda do you want to update test/images/agnhost/VERSION (for a new image to be built) in this PR itself?

Done

**Comment by AkihiroSuda** [comment]
Thanks for merging, but `post-kubernetes-push-e2e-agnhost-test-images` isn't working:

- https://github.com/kubernetes/kubernetes/issues/123266

**Pull request merged**